### PR TITLE
1.1 Colors

### PR DESCRIPTION
**Pull request by EgoziE**
Missing:
1. Description 
2. Colors that are not mapped
3. A thorough UX review + update zeplin

Colors story:
image

### CODE REVIEW

File: stories/Colors/colorsStory.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@import './src/colors';
```
Comment (shlomitc) [inline stories/Colors/colorsStory.scss]: You can just use the Box properties instead of a separate css file

File: stories/Colors/index.story.js [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+import { storySettings } from './storySettings';
+import { Layout, Cell } from '../../src/Layout';
+import classNames from 'classnames';
+import styles from './colorsStory.scss';
+import colors from '../../src/colors.scss';
+import Text from '../../src/Text';
+import Box from '../../src/Box';
+
+const empty = {};
+const lightText = true;
```
Comment (shlomitc) [inline stories/Colors/index.story.js]: Do you think we of a way to dynamically calculate this?

File: stories/Colors/index.story.js [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+const darkBorder = true;
+
+function rgba2hex(orig) {
```
Comment (TalZa1810) [inline stories/Colors/index.story.js]: Maybe we can add this util to the `Foundation` folder?
We have a similar util for calculating hex to rgba:
https://github.com/wix/wix-style-react/blob/master/src/Foundation/stylable/mixins/calc_rgba.js

**Comment by TalZa1810** [comment]
and also I think that we should consider adding a link in the cheatsheet `Foundation` section :)

**Comment by shlomitc** [comment]
@EgoziE - can we merge this?

**Comment by EgoziE** [comment]
It requires UX review first
I'll make sure it's done during sprint 11

File: stories/Colors/index.story.js [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+  },
+];
+
+export default {
+  category: storySettings.category,
+  storyName: storySettings.storyName,
+
+  sections: [
+    description({
+      title: '< Some Description >',
```
Comment (TalZa1810) [inline stories/Colors/index.story.js:316]: We should decide for a description here :)

File: stories/Introduction/Cheatsheet/componentsFamilies/FoundationFamily.js [hunk 5]
```diff
@@ -52,6 +52,20 @@ const TypographyDisplay = ({ children, label }) => (
...
+const ColorsExamples = () => {
+  const symbol = foundationSymbols.colors;
+
+  const singleComponentProps = {
+    name: createLinkedSymbolName({ groupSymbol: Category.FOUNDATION, symbol }),
+  };
+
+  return (
+    <SingleComponentSideBySide {...singleComponentProps}>
+      <Text>some description...</Text>
```
Comment (TalZa1810) [inline stories/Introduction/Cheatsheet/componentsFamilies/FoundationFamily.js:64]: We should decide for a description here :)

Comment (TalZa1810) [reply stories/Introduction/Cheatsheet/componentsFamilies/FoundationFamily.js:64]: also for a design for the example.

File: stories/Colors/index.story.js (hunk 2 above)
Comment (EgoziE) [reply stories/Colors/index.story.js:11]: https://jira.wixpress.com/browse/DSM-508
In a different PR

**Pull request merged**